swift/tests/CleanupStaleFlagsCommandInputValidatorTests.swift

```diff
@@ -87,6 +94,29 @@ final class CleanupStaleFlagsCommandInputValidatorTest: XCTestCase {
         }
     }
     
+    func test_validateInput_validFileURLs_invalidConfig_emptyFlagName() {
```

Comment:
additional tests to test: 
1. badly formed JSONs (e.g., just methodName)
2. badly formatted JSONs

Reply:
Yeah, makes sense. Added additional tests to cover the above.